### PR TITLE
[REF] Wallet Connect

### PR DESCRIPTION
**Pull request by gabrielbazan7**

### CODE REVIEW

File: app-template/copay/appConfig.json [hunk 1]
```diff
@@ -36,6 +36,6 @@
     "debitcard": false,
     "shapeshift": false,
     "giftcards": false,
-    "walletConnect": false
+    "walletConnect": true
```
Comment (cmgustavo) [inline app-template/copay/appConfig.json:39]: Are you going to support it on Copay app too?

Comment (gabrielbazan7) [reply app-template/copay/appConfig.json:39]: yess

File: src/providers/incoming-data/incoming-data.ts [hunk 2]
```diff
@@ -461,10 +461,10 @@ export class IncomingDataProvider {
 
   private handleWalletConnectUri(uri: string): void {
     // Disable Wallet Connect
-    if (!this.appProvider.info._enabledExtensions.walletConnect) {
-      this.logger.warn('Wallet Connect has been disabled for this build');
-      return;
-    }
+    // if (!this.appProvider.info._enabledExtensions.walletConnect) {
```
Comment (cmgustavo) [inline src/providers/incoming-data/incoming-data.ts:464]: I think it's necessary for disabling WC from appConfig

Comment (gabrielbazan7) [reply src/providers/incoming-data/incoming-data.ts:464]: +1

**Comment by Danilo-caio-da-silva-vieira** [comment]
Necessário nada

Dasilvadanilocaiodasilvavieira@gmail.com

Em qua, 23 de dez de 2020 11:49, Gustavo Maximiliano Cortez <
notifications@github.com> escreveu:

**Comment by cmgustavo** [comment]
"Disconnect" action should show an spinner or something to lock the screen while it's disconnecting.

Also I realized that after disconnect, I see the wallet to connect to (seems url or something is not refreshed). 

```
[debug] walletConnector.killSession
[debug] Storage Key: walletConnectSession removed
```

See screen when trying to re-connect after disconnect.

```
[info] Initialize wallet connect with uri: undefined
```

Screen Shot 2020-12-23 at 16 23 06

**Comment by cmgustavo** [comment]
"Sending from" should not be clickeable.

Screen Shot 2020-12-23 at 16 58 10

**Comment by cmgustavo** [comment]
Is there a way to know if I disconnected from the other side? I mean, disconnect "walletConnect" from Uniswap. It shows connected in the app. Is there a way to know if wallet is not linked?

**Comment by cmgustavo** [comment]
I think that we shouldn't log private info in console log: 
```
[debug] session {"connected":true,"accounts":..... 
```

**Comment by cmgustavo** [comment]
Transaction stuck on broadcasting. But transaction of 0.001 ETH has been sent (no contract). txid: `0xd698c8c2e41bb884dae7f22c211f706b3c0de41e4f5fa9d60d51402b2e1872b0`

IMG_1852

**Review by cmgustavo — DISMISSED** [review]
ACK! Great job! 🚀

**Review by cmgustavo — APPROVED** [review]

**Pull request merged**